plugins/forc-index/src/utils/defaults.rs

```diff
@@ -26,9 +26,9 @@ fuel-indexer-macros = {{ version = "0.9", default-features = false }}
 fuel-indexer-plugin = {{ version = "0.9", features = ["native-execution"] }}
 fuel-indexer-schema = {{ version = "0.9", default-features = false }}
 fuel-tx = "0.26"
-fuels = {{ version = "0.38.1" }}
-fuels-core = {{ version = "0.38.1" }}
-fuels-types = {{ version = "0.38.1", default-features = false }}
+fuels = {{ version = "0.40.0" }}
```

Comment:
@rvmelkonian Unrelated to your PR, but can we just use the minor version (and not the patch version) `0.40` here? I think it's safe to assume the latest version is OK to use

CC @deekerno for thoughts

Reply:
The Rust SDK team has already moved on to `v0.41.0`, so in _this_ particular case, I don't think it matters too much. But in general, yeah, I'd also like us to use the minor version as they're quite good at making sure that patches don't contain breaking changes.

Reply:
Noted, so we only want minor version bumps. I will test out the new `v0.41.0` after this one is merged.